heros/models.py

```diff
@@ -49,7 +49,6 @@ def __unicode__(self):
 class HeroRole(models.Model):
 
     ROLE_NAME_CHOICES = (
-        ('complexity', 'Complexity'),
```

Comment:
I know we are not using it but I would suggest we keep this option as its provide by dota in his game.